### PR TITLE
IIS Rewrite Maps and Redirecting On Url Paths

### PR DESCRIPTION
**Pull request by khalidabuhakmeh**

### CODE REVIEW

File: _posts/2019-07-26-iis-rewrite-maps-and-redirecting-on-url-paths.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+image_credit: darylgio agoncillo
+---
+
+URLs are the bridges that connect our little island of a website to the bigger continent known as the Internet. Preserving URL integrity is the utmost importance for providing an excellent experience for new and existing users alike. Sadly, from a development and management standpoint, keeping old paths working for a new site can be a real pain in the butt. In this post, we'll show you how to use redirect maps to create a lookup dictionary for old URL paths. This post will also help you preserve query string parameters.
+
+## Configuration
+
+We started by exploring the [Microsoft documentation site](https://docs.microsoft.com/en-us/iis/extensions/url-rewrite-module/using-rewrite-maps-in-url-rewrite-module) around rewrite maps. For rewrite maps, you will need to make sure you are running your site on Internet Information Services with the rewrite module.
+
+What you will in your web.config is a `rewriteMaps` section with all off your old URL paths and their new desitations.
```
Comment (khalidabuhakmeh) [inline _posts/2019-07-26-iis-rewrite-maps-and-redirecting-on-url-paths.md]:
```suggestion
What you will in your web.config is a `rewriteMaps` section with all of your old URL paths and their new destinations.
```

Comment (kendaleiv) [inline _posts/2019-07-26-iis-rewrite-maps-and-redirecting-on-url-paths.md]: Could you clarify *exactly* how to add these code sections in configuration? Basically, include `system.webServer`, `rules`, etc.

File: _posts/2019-07-26-iis-rewrite-maps-and-redirecting-on-url-paths.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <rewriteMaps>
+        <rewriteMap name="StaticRewrites" defaultValue="">
+            <add key="/one" value="/new-one" />
+            <add key="/two" value="/new-two" />
+        </rewriteMap>
+    </rewriteMaps>
+</rewrite>
+```
+
+This rewrite map then needs a subsequent rule to match the url path.
```
Comment (khalidabuhakmeh) [inline _posts/2019-07-26-iis-rewrite-maps-and-redirecting-on-url-paths.md]:
```suggestion
This rewrite map then needs a subsequent rule to match the URL path.
```

File: _posts/2019-07-26-iis-rewrite-maps-and-redirecting-on-url-paths.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+date: 2019-07-26 11:45:20
+tags:
+  - IIS
+  - Azure
+  - SEO
+categories:
+  - IIS
+  - Azure
+  - SEO
+twitter_text:
```
Comment (kendaleiv) [inline _posts/2019-07-26-iis-rewrite-maps-and-redirecting-on-url-paths.md]: Want to add something here?

**Pull request merged**